index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title>Acme Pets - Products</title>
+	<link rel="stylesheet" href="products.css">
+</head>
+<body>
+	<h1 class="heading">Dog Food</h1>
+		<div id="table-container" class="table-container">
+		</div>
+	<h1 class="heading">Cat Food</h1>
+		<div id="cat-container" class="table-container">
+		</div>
```

Comment:
I'd simplify your divs to `<div id="table-container" class="table-container"></div>`